Github Actions failure on Windows

Checking the [actions](https://github.com/google/pprof/actions?query=workflow%3Aci) tab, it looks like there are failures on the main branch starting about 7 days ago. 

This seems to be the failure (from https://github.com/google/pprof/runs/1483842328?check_suite_focus=true):
```
=== RUN   TestWebInterface
Format: "svg" not recognized. Use one of:
    proftest.go:125: unexpected error: Failed to execute dot. Is Graphviz installed?
        exit status 1
    webui_test.go:105: response for / does not match expected pattern 'F1'; actual result:
        Could not execute dot; may need to install graphviz.
    webui_test.go:105: response for / does not match expected pattern 'F2'; actual result:
        Could not execute dot; may need to install graphviz.
    webui_test.go:105: response for / does not match expected pattern 'F3'; actual result:
        Could not execute dot; may need to install graphviz.
    webui_test.go:105: response for / does not match expected pattern 'testbin'; actual result:
        Could not execute dot; may need to install graphviz.
    webui_test.go:105: response for / does not match expected pattern 'cpu'; actual result:
        Could not execute dot; may need to install graphviz.
Format: "svg" not recognized. Use one of:
Format: "svg" not recognized. Use one of:
    proftest.go:125: unexpected error: Failed to execute dot. Is Graphviz installed?
        exit status 1
    proftest.go:125: unexpected error: Failed to execute dot. Is Graphviz installed?
        exit status 1
--- FAIL: TestWebInterface (0.19s)
```

Base on [the last passing run](https://github.com/google/pprof/runs/1445501643?check_suite_focus=true), this test was previously skipped on Windows. 

```
=== RUN   TestWebInterface
    webui_test.go:88: skipping / since dot (graphviz) does not seem to be installed
```

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -135,6 +135,11 @@ jobs:
 
       - name: Run the test
         run: |
+          # This is a workaround to make graphviz installed through choco work.
+          # It generates a config file to tell dot what layout engine and 
+          # format types are available. See
+          # https://github.com/google/pprof/issues/585 for more details.
+          dot -c
           go env
           go build github.com/google/pprof
           go test -v ./...

